Evaluation/Input/Content/titlepage.md

```diff
@@ -2,11 +2,11 @@
 
 
 
-| Version                                         | 1.0-OSP9.1                                                   |
+| Version                                         | x.x-OSPy.y                                                   |
 | ----------------------------------------------- | ------------------------------------------------------------ |
 | based on *Model Snapshot* and *Evaluation Plan* | https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Model/releases/tag/v1.0 |
```

Comment:
`https://github.com/Open-Systems-Pharmacology/Fluvoxamine-Model/releases/tag/v1.0`
Either replace `v1.0` with `vx.x` - or it must be 1.2(?) (and then x.x above could also be 1.2)

In general I think it is better to replace **x.x** with the next release version of the model (which is known before you make a new release in a model repository). We had already a short discussion on this as far as I remember @sfrechen 
The drawback of this proposal: I'm afraid the model developers will often forget to increment this version in the report.
For a reviewer later is then not obvious, if the version is correct (while having "x.x" is obviously wrong in the final report and needs to be replaced)

Reply:
Changed to `vx.x`.